### DISCUSSION TITLE
how can i invoke cuda kernal from golang code

### DISCUSSION THREAD

**Discussion by hunjixin** [JuliaGPU/CUDA.jl#2420] [Q&A] [answered]
if i have a kernel writen with julia,  how can i invoke with golang code

**Comment by maleadt** [comment, marked as answer]
Since there doesn't seem to be a Go<->Julia integration package, you can use the Julia C API, https://docs.julialang.org/en/v1/manual/embedding/, and call that using Go's C FFI.
